README.md

```diff
@@ -0,0 +1,130 @@
+<div align="center">
+  <h3><b>Hello-Rails-Backend README</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    <!-- - [Key Features](#key-features) -->
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  <!-- - [Run tests](#run-tests) -->
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+<!-- - [🙏 Acknowledgements](#acknowledgements) -->
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+# 📖 Hello-Rails-Backend  <a name="about-project"></a>
+
+This app is all about building rails Api.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+> ### FrontEnd PR
+
+[react pull request link](https://github.com/Fayob/hello-react-front-end/pull/1)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A text editor preferably [Visual Studio code](https://code.visualstudio.com/)
+- Latest version of [Ruby](https://www.ruby-lang.org/en/downloads/) installed
+- [PostgreSQL Server](https://www.postgresql.org/download/)
+
+
+### Setup
+
+Clone this [repository](https://github.com/Fayob/hello-rails-back-end) to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Fayob/hello-rails-back-end.git
+  cd hello-rails-back-end
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+
+```sh
+  rails s
+```
+
+Open your browser and go to "http://localhost:3000/" to see the working project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I highly recommend to fix the back to top link since it is brokend and when I press on it I can't see that it took me to the top ✨